libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -111,6 +111,20 @@ static bool _createKeepAliveOperation( const IotMqttNetworkInfo_t * pNetworkInfo
                                        uint16_t keepAliveSeconds,
                                        _mqttConnection_t * pMqttConnection );
 
+/**
+ * @brief Initialize a network connection, creating it if necessary.
+ *
+ * @param[in] pNetworkInfo User-provided network information for the connection
+ * connection.
+ * @param[out] pNetworkConnection On success, the created and/or initialized network connection.
+ * @param[out] pCreatedNetworkConnection On success, `true` if the network connection was created; `false` if it was only initialized.
+ *
+ * @return Any #IotNetworkError_t, as defined by the network stack.
```

Comment:
These comments seems less clear to me now that the function is renamed. Perhaps rewording them to clarify the difference between initializing and creating, e.g. "true if a new network connection was created; false if an existing one will be used"

Reply:
I was able to eliminate this parameter, agreed that it is confusing.

Reply:
Removing this parameter has caused more problems than it fixed, so I've restored it.  Will update documentation as suggested.